dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,153 @@
+###############################################################################
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMCM
+#COMPONENTNAMEFOR CWMCM MCP Server
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+
+# Used for server logging
+# do not translate MCP
+# {0} = The qualified name of the tool
+CWMCM0001E.blank.arguments=CWMCM0001E: The {0} MCP tool method has one or more arguments with blank names
+CWMCM0001E.blank.arguments.explanation=MCP Tool arguments were provided but they had missing names. Each argument to a tool method must have a name.
+CWMCM0001E.blank.arguments.useraction=Provide non-empty values for all required arguments.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The  qualified name of the tool
+# {1} = corresponding argument name
+CWMCM0002E.duplicate.arguments=CWMCM0002E: The {0} MCP tool method has more than one argument named {1}
```

Comment:
Add a period to the end of the sentence unless the style for this message type is not to end with a period or unless the {1} variable would be confusing to read with a period after it.